m2m-transfers/V1/m2m-transfers-json-contract.md

```diff
@@ -332,19 +332,53 @@ EDITTRAN - Edit existing transaction (expire existing transfer & create a new tr
 }
 ```
 ### Error Codes
-501 - Config file read / validation error
-502 - Invalid source account
-503 - Invalid recipient account
-504 - Insufficient information
-505 - Invalid input
-506 - Error creating/deleting recipient
-507 - Error creating/updating/deleting transfer
-508 - Undefined error
-509 - Member unverified
-510 - AccountId incorrect
+
+*See Modifier section for additional details
+
+| Error Code | Logging Error Message                                                                 | Modifier                                                                             | Additional Notes As Needed                    |
+| ---------- | ------------------------------------------------------------------------------------- |------------------------------------------------------------------------------------- |-----------------------------------------------|
+| 500        | Program running in memo mode |||
+| 501        | Config file error | : [configuration file name] open error - [system generated letter file read error message] ||
+|            || : [configuration file name] read error - [system generated letter file read error message] ||
+|            || : 3-Error Reading Letterfile [configuration file name]: [system generated letter file read error message] ||
+|            || : Invalid Parameter in CFG file ||
+| 502        | Invalid Source Account | : Pref Access type 3 not found ||
+|            || : Acct Warning 1234 ||
+|            || : No eligible transfer from shares/loans ||
+|            || : Acct Type 1234 ||
+| 503        | Invalid Recipient Account |||
+| 504        | Invalid Source Account | Cannot calc member limits. ||
```

Comment:
We would need to change this in code also but I think, for consistence, the modifier text should be prefixed by ":"

Reply:
![image](https://github.com/Banno/banno-powerons/assets/97116592/2e79ee51-4de9-4383-913f-cef611e237cd)

** **Same comment applies to V3 files**

Reply:
I've moved it up for Error code 509 as I verified that the code will always have a specific modifier.

However, for Error Code 507 to the Modifier section for the generic error, I've added comment ' No Modifier. Error Code 507 does not always provide a Modifier.’.